### PR TITLE
[CLI-FEATURE] add registries options for oc publish

### PR DESCRIPTION
**Pull request by ricardo-devis-agullo**
Right now, to publish to registries you need to have set a `oc.json` file with your registries. This works well when you only publish to one registry, or you always publish to all the registries at the same time (so you can have your oc.json versioned in your repo and just use it).

The problem is that that solution doesn't work as well when your deployment setup is to publish to a different registry depending on your environment (1 or more registries on each environment). In that case, for every deployment, you have to go and edit the oc.json before being able to call `oc publish`.

For those cases, I'm adding a new option `--registries` that allows you to pass a list of registries that will override your oc.json configuration. So when you want to publish you can just do

```
oc publish your-component --registries http://www.myregistry.com https://www.myregistry2.com
```
without needing a oc.json

### CODE REVIEW

File: src/cli/commands.ts [hunk 1]
```diff
@@ -128,6 +128,11 @@ export default {
           boolean: true,
           description: 'Skip packaging step',
           default: false
+        },
+        registries: {
+          array: true,
+          description:
+            'registries to publish, overriding your configuration in oc.json'
```
Comment (pbazydlo) [inline src/cli/commands.ts:135]: `List of registries to publish to. This setting will take precedence over oc.json file.`

Comment (ricardo-devis-agullo) [reply src/cli/commands.ts]: done

File: test/unit/cli-facade-publish.js [hunk 2]
```diff
@@ -101,6 +102,20 @@ describe('cli : facade : publish', () => {
         });
       });
 
+      it('should take precedence the registries set through options', done => {
```
Comment (pbazydlo) [inline test/unit/cli-facade-publish.js:105]: `should take precedence over the...`

Comment (ricardo-devis-agullo) [reply test/unit/cli-facade-publish.js]: done

**Pull request merged**